chore: move button into view and style files

J-Son89 opened the pull request:
This pr is related to this issue: https://github.com/status-im/status-mobile/issues/16396

It's part 1 of some changes that I will be making to refactor the button component, the work looks quite large and this will hopefully make it easier to review the follow up prs being made 👍 

Testing:
nothing is actually changed here, I just moved the file structure of the buttons and updated some imports to require from quo2.core rather than the button file directly.

J-Son89 committed:
chore: move button into view and style files

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 26e9de88 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-16442/1/) | 2023-06-29 12:31:09 | ~6 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-230629-122503-26e9de-pr16442-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-230629-122503-26e9de-pr16442-universal.apk)|

J-Son89 committed:
chore: adjust button imports to be from quo2.core

J-Son89 commented on src/quo2/components/buttons/button/component_spec.cljs:
```diff
@@ -0,0 +1,25 @@
+(ns quo2.components.buttons.button.component-spec
```
the only actual changes in this pr is that I used the test helpers for this test rather the old direct methods. However the tests are not changed at all 👍

J-Son89 commented:
@churik - do you think this pr needs QA review? - nothing is actually changed here, I just moved the file structure of the buttons and updated some imports to require from quo2.core rather than the button file directly.
Let me know what you think!

status-im-auto commented:
## 94% of end-end tests have passed
```
Total executed tests: 34
Failed tests: 2
Passed tests: 32
```
```
IDs of failed tests: 702732,702731 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=16442&TR_CASE_IDS=702732,702731">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185498">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185499">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (32)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185528">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/701407c838194e259df4fd8ccb61b76f?auth=b790fee6c1a9d67d9623e20ae7f91da8#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185501">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/701407c838194e259df4fd8ccb61b76f?auth=b790fee6c1a9d67d9623e20ae7f91da8#92">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185517">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/701407c838194e259df4fd8ccb61b76f?auth=b790fee6c1a9d67d9623e20ae7f91da8#74">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185503">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185500">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#320">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#259">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185505">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#409">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#454">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185509">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#251">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#191">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185502">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#190">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#147">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185504">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#374">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#325">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185520">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#294">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#218">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185497">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/24aeb717829a400bb5182d3103f8f22f?auth=ed76f705e731d2197e5e076e1c4a2ab3#76">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77d32600e90247ffb94d53a77c2049e4?auth=fd08ebfb8caae3e64ab17cdcc911832c#73">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185530">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f162f4653841c799bce13086fd794e?auth=2f18819aa2a147a059b0e442bb87ff43#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a9babd2dafcf4f0cbe03cc87840f5377?auth=b5cf4f47586661768473888381ebf100#92">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3c9b0c1e79d4069b5f457c49c904306?auth=497945dafa42d21d6dc38d23aab49181#72">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185507">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f162f4653841c799bce13086fd794e?auth=2f18819aa2a147a059b0e442bb87ff43#121">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a9babd2dafcf4f0cbe03cc87840f5377?auth=b5cf4f47586661768473888381ebf100#67">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3c9b0c1e79d4069b5f457c49c904306?auth=497945dafa42d21d6dc38d23aab49181#67">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185508">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f162f4653841c799bce13086fd794e?auth=2f18819aa2a147a059b0e442bb87ff43#236">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a9babd2dafcf4f0cbe03cc87840f5377?auth=b5cf4f47586661768473888381ebf100#146">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3c9b0c1e79d4069b5f457c49c904306?auth=497945dafa42d21d6dc38d23aab49181#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185519">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c3436af2be64a44833bfdbbefc06942?auth=f36835a589aaa41245943f66f4b99232#88">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc04c5b9a95c4118923ad0e2155c6aba?auth=822085e8584c954376222f6893a3b566#92">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185518">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c3436af2be64a44833bfdbbefc06942?auth=f36835a589aaa41245943f66f4b99232#55">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc04c5b9a95c4118923ad0e2155c6aba?auth=822085e8584c954376222f6893a3b566#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185529">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#233">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#194">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185521">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#207">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#156">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185512">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#254">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#231">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185527">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#466">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#351">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185522">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#339">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185506">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#485">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#374">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185511">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#186">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#127">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185510">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#144">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185515">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#275">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#246">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185514">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#162">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185516">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#498">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#397">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185513">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fcdb91dee164a0ebc325926102b8201?auth=d75beb352a050f83c2e4a7608cd04b63#314">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72b0e90e361a4d1093a6eabe6da0a0ab?auth=0c2af42e238beca6d399ec6f42df61e6#283">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185525">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/047f08ebd42a4b3cb095f59157b2dc04?auth=196aeb20ddd7cea7c6afeb8cb9ca0200#275">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6301479b2cc74a068b740b0254f3147e?auth=b9899d186e33e9204a624e79145ed164#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185523">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/047f08ebd42a4b3cb095f59157b2dc04?auth=196aeb20ddd7cea7c6afeb8cb9ca0200#142">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6301479b2cc74a068b740b0254f3147e?auth=b9899d186e33e9204a624e79145ed164#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185524">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/047f08ebd42a4b3cb095f59157b2dc04?auth=196aeb20ddd7cea7c6afeb8cb9ca0200#163">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6301479b2cc74a068b740b0254f3147e?auth=b9899d186e33e9204a624e79145ed164#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3185526">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/047f08ebd42a4b3cb095f59157b2dc04?auth=196aeb20ddd7cea7c6afeb8cb9ca0200#311">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6301479b2cc74a068b740b0254f3147e?auth=b9899d186e33e9204a624e79145ed164#140">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik commented:
no, go ahead, thanks for heads-up! @J-Son89
